qa-tests-backend/src/test/groovy/IntegrationsSplunkViolationsTest.groovy

```diff
@@ -101,7 +99,8 @@ class IntegrationsSplunkViolationsTest extends BaseSpecification {
                 ["name": SPLUNK_INPUT_NAME, "interval": "1", "from_checkpoint": "2000-01-01T00:00:00.000Z"])
     }
 
-    @Category([Integration, BAT])
+    @Tag("Integration")
+    @Tag("BAT")
```

Comment:
probably doesn't matter, but typically, I have seen BAT first

Reply:
> There are some issues with building images on OSCI. I added `turbo-build` label to build it on GHA

Thanks! That got the images built in time for the e2e tests 🕺